cse-lib/network-processing/src/main/java/com/farao_community/farao/cse/network_processing/busbar_change/BusBarChangePreProcessor.java

```diff
@@ -42,21 +43,44 @@ private BusBarChangePreProcessor() {
         // utility class
     }
 
+    private static CRACDocumentType importNativeCrac(InputStream inputStream) {
```

Comment:
Duplicate from export_runner's `FileImporter`.

Reply:
As discussed, this point should be handled in a separate ticket dedicated to refactoring. Won't be changed in this PR.